changelog/6735.misc.md

```diff
@@ -0,0 +1,3 @@
+Update tests to investigate `test_lock_store.py` behavior.

```

Comment:
I would add a sentence about how this has been updated. Something like 
```suggestion
Update tests in `test_lock_store.py`. Some of these tests are now allowed 
to fail on Windows and will be investigated further.
```

Reply:
It's not gonna be added to changelog anyway.